app/uk/gov/hmrc/essttpstubs/services/EligibilityService.scala

```diff
@@ -34,16 +34,8 @@ class EligibilityService @Inject()(ttp: ActorRef[Command])(implicit S: Scheduler
 
   implicit val timeout = Timeout(5.seconds)
 
-  def error(regime: TaxRegime, id: TaxID, error: EligibilityError): SR[Unit] = {
-    EitherT(ttp.ask(ref => MapPayeError(id, error,ref)))
-  }
-
-  def financials(regime: TaxRegime, id: TaxID, data: OverduePayments): SR[Unit] = {
-    EitherT(ttp.ask(ref => MapFinancialData(id, data,ref)))
-  }
-
-  def eligibilityData(regime: TaxRegime, id: TaxID): SR[OverduePayments] = {
-    EitherT(ttp.ask(ref => FindEligibilityData(id,ref)))
+  def eligibilityData(regime: TaxRegime, id: TaxID): Future[TTPEligibilityData] = {
+    ttp.ask(ref => FindEligibilityData(id,ref))
```

Comment:
I woudn't use any actors here, but since this is Stub we can go wild and develop it in more fancy ways :)